src/gbd_mapping_generator/data.py

```diff
@@ -206,9 +206,9 @@ def get_age_restriction_edge(age_restriction, end=False):
     id_map = {
         0.0: [2, None],
         0.01: [3, 2],
-        0.10: [4, 3],
-        1.0: [5, 4],
-        5.0: [6, 5],
+        0.10: [388, 3],
+        1.0: [238, 389],
+        5.0: [6, 34],
```

Comment:
I think this should contain all possible edges, not just the ones you actually see, so you should add an entry for each of the boundaries between the new age bins as well.